qualibration_libs/plotting/standard_plotter.py

```diff
@@ -0,0 +1,196 @@
+from typing import Dict, List, Literal, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import plotly.graph_objects as go
+import xarray as xr
+from matplotlib.figure import Figure as MatplotlibFigure
+from plotly.subplots import make_subplots
+from pydantic import BaseModel, Field
+from qualibration_libs.analysis import lorentzian_dip
+from qualibration_libs.plotting.grids import (PlotlyQubitGrid, QubitGrid,
+                                              grid_iter, plotly_grid_iter)
+from quam_builder.architecture.superconducting.qubit import AnyTransmon
+
+
+class TraceConfig(BaseModel):
```

Comment:
This needs a docstring explaining its purpose. The same applies to the classes below.